src/addons/grids/ReactDataGrid.js

```diff
@@ -117,6 +117,12 @@ var ReactDataGrid = React.createClass({
     if(nextProps.rowsCount  === this.props.rowsCount + 1){
       this.onAfterAddRow(nextProps.rowsCount + 1);
     }
+    if(nextProps.sortColumn){
+      this.setState({ sortColumn: nextProps.sortColumn });
+    }
+    if(nextProps.sortDirection){
+      this.setState({ sortDirection: nextProps.sortDirection });
```

Comment:
hey @alistairjcbrown, looks good but one thing i'll say is it might be good to include either `nextProps.sortDirection` or a default and only call `setState()` once with both properties specified?

please can you add some tests and then we'll get this merged :smile: